sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/enums/LightName.java

```diff
@@ -0,0 +1,134 @@
+package com.smartdevicelink.proxy.rpc.enums;
+
+public enum LightName {
+	/*Common Single Light*/
+	FRONT_LEFT_HIGH_BEAM,
+	FRONT_RIGHT_HIGH_BEAM,
+	FRONT_LEFT_LOW_BEAM,
+	FRONT_RIGHT_LOW_BEAM,
+	FRONT_LEFT_PARKING_LIGHT,
+	FRONT_RIGHT_PARKING_LIGHT,
+	FRONT_LEFT_FOG_LIGHT,
+	FRONT_RIGHT_FOG_LIGHT,
+	FRONT_LEFT_DAYTIME_RUNNING_LIGHT,
+	FRONT_RIGHT_DAYTIME_RUNNING_LIGHT,
+	FRONT_LEFT_TURN_LIGHT,
+	FRONT_RIGHT_TURN_LIGHT,
+	REAR_LEFT_FOG_LIGHT,
+	REAR_RIGHT_FOG_LIGHT,
+	REAR_LEFT_TAIL_LIGHT,
+	REAR_RIGHT_TAIL_LIGHT,
+	REAR_LEFT_BREAK_LIGHT,
```

Comment:
These now need to be updated per a proposal update to REAR_LEFT_BRAKE_LIGHT and REAR_RIGHT_BRAKE_LIGHT